### PR TITLE
Add a section about flotta-dev-cli to the developer guide

### PR DESCRIPTION
**Pull request by arielireni**
Signed-off-by: arielireni <aireni@redhat.com>

### CODE REVIEW

**Comment by arielireni** [comment]
@eloycoto  @jakub-dzon Can you please review this PR?

**Review by jakub-dzon — APPROVED** [review]

File: documentation/v0_2_0/dev/guide.md [hunk 1]
```diff
@@ -148,3 +148,27 @@ machinectl shell -q flotta@.host /usr/bin/journalctl --user
 - *Go/Lint*: Checks that code is correctly linted
 - *Go/SecurityScanner*: Checks that the current code is statically checked by
   [gosec](https://github.com/securego/gosec)
+
+# Developer CLI
+
+### Prerequisites
+- Install the flotta operator as mentioned [here](https://project-flotta.io/documentation/v0_2_0/dev/installation.html).
```
Comment (pkliczewski) [inline documentation/v0_2_0/dev/guide.md:155]: What do you think about using `https://project-flotta.io/documentation/latest/dev/installation.html`?

**Review by pkliczewski — COMMENTED** [review]
@arielireni I would update the post for latest first and later for other versions.

File: documentation/v0_2_0/dev/guide.md (hunk 1 above)
Comment (arielireni) [reply documentation/v0_2_0/dev/guide.md]: Done :)

**Pull request merged**